src/requests/actions.jl

```diff
@@ -75,11 +75,11 @@ function textDocument_codeAction_request(params::CodeActionParams, server::Langu
 end
 
 function workspace_executeCommand_request(params::ExecuteCommandParams, server::LanguageServerInstance, conn)
-    uri = params.arguments[1]
-    offset = params.arguments[2]
-    doc = getdocument(server, uri)
-    x = get_expr(getcst(doc), offset)
     if haskey(LSActions, params.command)
+        uri = URI(params.arguments[1])
+        offset = params.arguments[2]
+        doc = getdocument(server, uri)
+        x = get_expr(getcst(doc), offset)
```

Comment:
I've moved the uri/offset extraction into the `if` block because that "calling convention" is only valid for actions we define ourselves, and who knows what a client might send.